Source/monster.cpp

```diff
@@ -4773,18 +4765,18 @@ bool LineClear(bool (*Clear)(int, Point), int entity, int x1, int y1, int x2, in
 	int tmp;
 	bool done = false;
 
-	xorg = x1;
-	yorg = y1;
-	dx = x2 - x1;
-	dy = y2 - y1;
+	xorg = startPoint.x;
+	yorg = startPoint.y;

```

Comment:
A few things in this function could benefit from xorg/yorg becomming `Point originalStart`.

Reply:
`startPoint` might also be better named as just point or position as it seams to be changed over time. Then xorg/yorg could instead be called `startPoint`.